src/remote_connector_dao/test_real_time_graph.py

```diff
@@ -14,9 +18,16 @@ def run_real_time_graph():
         timetagger_proxy, tagger_controller, single_channels_list
     )
 
+    # signal_counter = SignalCounter()
+
     channels = [*single_channels_list, coincidences_virtual_channel.getChannels()[0]]
 
     for channel in single_channels_list:
         tagger_controller.setTriggerLevel(channel, TRIGGER_VOLTAGE)
 
-    plot_real_time_graph(timetagger_proxy, tagger_controller, channels)
+    app = QtWidgets.QApplication(sys.argv)
```

Comment:
Acho que podemos mover a criacao do app para o `MainWindow`, ja que ambos sao usados juntos

Reply:
Pelos exemplos que estava estudando, o app é criado na função que instancia a  `MainWindow`, geralmente chamam essa função de `main()`.

Reply:
Mas os exemplos tem algum motivo para isso? Imagino que se o `app` nao for usado em outros lugares que fora da `MainWindow`, faria sentido colocar eles associados.

Eu chuto que, quando o app abre fechado/em background, faz sentido ter app fora do `MainWindow`, pois o `MainWindow` so seria aberto depois de clicar no `TrayIcon`.

Mas nao sei, estou apenas divagando. Nao acho que eh uma mudanca requerida